superset/db_engine_specs/presto.py

```diff
@@ -794,13 +794,14 @@ def extra_table_metadata(cls, database, table_name, schema_name):
         if schema_name and "." not in table_name:
             full_table_name = "{}.{}".format(schema_name, table_name)
         pql = cls._partition_query(full_table_name)
-        col_name, latest_part = cls.latest_partition(
+        col_names, latest_parts = cls.latest_partition(
             table_name, schema_name, database, show_first=True
         )
+        latest = {col_name: latest_parts[i] for i, col_name in enumerate(col_names)}
```

Comment:
You can use `zip` here, I think it expresses intent more clearly:

```python
latest = {
    col_name: latest_part
    for col_name, latest_part in zip(col_names, latest_parts)
}
```

Or even:

```python
latest = dict(zip(col_names, latest_parts)))
```